test/functional/sqlsrv/0052.phpt

```diff
@@ -1,31 +1,32 @@
 --TEST--
 scrollable results with no rows.
+--DESCRIPTION--
+this test is very similar to test_scrollable.phpt... might consider removing this test as a duplicate
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_versions_old.inc'); ?>
 --FILE--
 <?php
     sqlsrv_configure('WarningsReturnAsErrors', 0);
     sqlsrv_configure('LogSeverity', SQLSRV_LOG_SEVERITY_ALL);
 
     require_once('MsCommon.inc');
 
-    $conn = connect();
-    if ($conn === false) {
-        die(print_r(sqlsrv_errors(), true));
-    }
+    $conn = AE\connect();
+    $tableName = 'ScrollTest';
 
-    $stmt = sqlsrv_query($conn, "IF OBJECT_ID('ScrollTest', 'U') IS NOT NULL DROP TABLE ScrollTest");
-    if ($stmt !== false) {
-        sqlsrv_free_stmt($stmt);
-    }
+    $columns = array(new AE\ColumnMeta('int', 'id'),
+                     new AE\ColumnMeta('char(10)', 'value'));
+    $stmt = AE\createTable($conn, $tableName, $columns);
+    sqlsrv_free_stmt($stmt);
 
-    $stmt = sqlsrv_query($conn, "CREATE TABLE ScrollTest (id int, value char(10))");
-    if ($stmt === false) {
-        die(print_r(sqlsrv_errors(), true));
+    $query = "SELECT * FROM $tableName";
+    if (AE\isColEncrypted()) {
+        $options = array('Scrollable' => SQLSRV_CURSOR_FORWARD);
```

Comment:
please include a comment that says Column Encryption does not support scrollable cursor other than SQLSRV_CURSOR_FORWARD